SpatialGDK/Build/Programs/Improbable.Unreal.Scripts/DeploymentLauncher/DeploymentLauncher.cs

```diff
@@ -462,20 +473,20 @@ private static void ShowUsage()
         {
             Console.WriteLine("Usage:");
             Console.WriteLine("DeploymentLauncher create <project-name> <assembly-name> <main-deployment-name> <main-deployment-json> <main-deployment-snapshot> <main-deployment-region> [<sim-deployment-name> <sim-deployment-json> <sim-deployment-region> <num-sim-players>]");
-            Console.WriteLine($"  Starts a cloud deployment, with optionally a simulated player deployment. The deployments can be started in different regions ('EU', 'US' and 'AP').");
-            Console.WriteLine("DeploymentLauncher stop <project-name> [deployment-id]");
+            Console.WriteLine($"  Starts a cloud deployment, with optionally a simulated player deployment. The deployments can be started in different regions ('EU', 'US' , 'AP' and 'CN').");
+            Console.WriteLine("DeploymentLauncher stop <project-name> <main-deployment-region> [deployment-id]");
             Console.WriteLine("  Stops the specified deployment within the project.");
             Console.WriteLine("  If no deployment id argument is specified, all active deployments started by the deployment launcher in the project will be stopped.");
-            Console.WriteLine("DeploymentLauncher list <project-name>");
+            Console.WriteLine("DeploymentLauncher list <project-name> <main-deployment-region>");
             Console.WriteLine("  Lists all active deployments within the specified project that are started by the deployment launcher.");
         }
 
         private static int Main(string[] args)
         {
             if (args.Length == 0 ||
                 args[0] == "create" && (args.Length != 11 && args.Length != 7) ||
```

Comment:
We don't actually use stop or list - there is code for stop but it's not bound to any UI in UE